### PR TITLE
 gpu: add stretched mode support 

### LINKED ISSUE #6487: Stretched mode plugin does not work with gpu plugin enabled
**Describe the bug**
When having both the gpu plugin and stretched mode plugin enabled in either fixed or stretched mode it will glitch out and malform your game window.

**To Reproduce**
1. Enable both the gpu plugin and stretched mode plugin
2. See what happens

**Expected behavior**
I would expect them to work at the same time

**Screenshots**
https://gyazo.com/b6f309eaafda7a7bb06eee7ca8ee6971

**Environment (please complete the following information):**
 - OS: [Windows]

### PR DESCRIPTION
**Pull request by devLotto**
Draws the scene + ui on a frame buffer that is then stretched when it's drawn on the default framebuffer.

Maybe there is a more clever way but this seems like it works, on my system anyway.

Fixes #6487

### CODE REVIEW

**Comment by formatme** [comment]
https://i.imgur.com/J4ktrBL.png
everything stops rendering on screen but works

**Comment by devLotto** [comment]
@formatme does #6550 work for you?

**Comment by MP2E** [comment]
This works well on my system. The UI did disappear for me when I tried to merge both the anti aliasing PR and this one together, but that was fixed by simply disabling antialiasing if stretched mode is turned on by changing

```
if (aaEnabled)
```

to

```
if(aaEnabled && !client.isStretchedEnabled())
```

in both locations where aaEnabled is checked. Not sure if that's the best solution in general

**Comment by devLotto** [comment]
Yeah you can't use both at the same time without changing some code. Were you doing that? @formatme

File: runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java [hunk 1]
```diff
@@ -679,6 +736,29 @@ public void draw()
...
+			if (lastStretchedDimensions == null || !lastStretchedDimensions.equals(stretchedDimensions))
+			{
+				shutdownStretchedFbo();
+				initStretchedFbo(stretchedDimensions.width, stretchedDimensions.height);
+				lastStretchedDimensions = stretchedDimensions;
+			}
+
+			gl.glBindFramebuffer(gl.GL_DRAW_FRAMEBUFFER, fboStretchedHandle);
+		}
+		else
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java]: `else if (lastStretchedDimensions != null)`

**Comment by formatme** [comment]
ya i was trying to use both.

**Comment by formatme** [comment]
thanks MP2E

**Pull request merged**